lib/tests/formulae.rb

```diff
@@ -216,6 +216,16 @@ def bottle_reinstall_formula(formula, new_formula, args:)
         test "brew", "install", @bottle_filename
       end
 
+      def all_bottled_deps?(formula)
```

Comment:
I probably want to call this `all_deps_bottled?` instead. `all_bottled_deps?` or `deps_all_bottled?` makes me think of all dependencies having `all` bottles.